crates/chronicle/src/bootstrap/cli.rs

```diff
@@ -1050,7 +1050,8 @@ impl SubCommand for CliModel {
                         .help("which API endpoints to offer")
                     ),
             )
-            .subcommand(Command::new("verify-keystore").about("Initialize and verify keystore, then exit"));
+            .subcommand(Command::new("verify-keystore").about("Initialize and verify keystore, then exit"))
+            .subcommand(Command::new("import").about("Import and apply Chronicle operations, then exit").arg(Arg::new("namespace-id").value_name("namespace-id").required(true).help("ExternalId of the Namespace to import into")).arg(Arg::new("namespace-uuid").value_name("namespace-uuid").required(true).help("UUID of the Namespace to import into")).arg(Arg::new("path").value_name("path").value_hint(ValueHint::FilePath).default_value("import.json").help("Path to data import file")));
```

Comment:
I think I've only ever seen default filepaths overridden with named arguments, I don't think I've ever seen this style of an unnamed optional filepath argument defaulting to something other than standard input. Not sure that unusual means bad though, just raising the question.

Reply:
(I've wondered, why isn't fmt pickier about this CLI stuff? That's a very long line.)

Reply:
> I think I've only ever seen default filepaths overridden with named arguments, I don't think I've ever seen this style of an unnamed optional filepath argument defaulting to something other than standard input. Not sure that unusual means bad though, just raising the question.

I love that suggestion. I've changed things so that there is no default path and the `PATH` is required unless the user selects the `--stdin` flag.